utilities/after_success.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This script is used by Travis-CI to publish artifacts (binary, sorce and javadoc jars) when releasing snapshots.
+# This script is referenced in .travis.yml.
+
+echo "Travis branch:       " ${TRAVIS_BRANCH}
+echo "Travis pull request: " ${TRAVIS_PULL_REQUEST}
+echo "Travis JDK version:  " ${TRAVIS_JDK_VERSION}
+if [ "${TRAVIS_JDK_VERSION}" == "oraclejdk7" -a "${TRAVIS_BRANCH}" == "master" -a "${TRAVIS_PULL_REQUEST}" == "false" ]; then
+    mvn cobertura:cobertura coveralls:report
+    mvn site-deploy -DskipTests=true --settings=target/travis/settings.xml
+    mvn deploy -DskipTests=true -Dgpg.skip=true --settings target/travis/settings.xml
```

Comment:
credentials are provided in the secure section of the [.travis.yaml](https://github.com/GoogleCloudPlatform/gcloud-java/blob/master/.travis.yml#L21).